activesupport/lib/active_support/core_ext/string/output_safety.rb

```diff
@@ -12,7 +12,7 @@ module Util
     JSON_ESCAPE_REGEXP = /[\u2028\u2029&><]/u
 
     # Following XML requirements: https://www.w3.org/TR/REC-xml/#NT-Name
-    TAG_NAME_START_REGEXP_SET = ":A-Z_a-z\u{C0}-\u{D6}\u{D8}-\u{F6}\u{F8}-\u{2FF}\u{370}-\u{37D}\u{37F}-\u{1FFF}" \
+    TAG_NAME_START_REGEXP_SET = "@:A-Z_a-z\u{C0}-\u{D6}\u{D8}-\u{F6}\u{F8}-\u{2FF}\u{370}-\u{37D}\u{37F}-\u{1FFF}" \
```

Comment:
Is there a different spec that says we should allow `@`?  The linked spec in the comment definitely doesn't allow `@`

Reply:
Never mind, I found the spec [here](https://html.spec.whatwg.org/multipage/syntax.html#attributes-2)